src/components/calculator.js

```diff
@@ -0,0 +1,21 @@
+import './calculator.css';
+import Display from './display';
+import BtnContainer from './btnContainer';
+
+function CalcContainer() {
+  return (
+    <div className="calcContainer">
+      <Display />
+      <BtnContainer />
+    </div>
+  );
+}
+
+// const CalcContainer = () => (
+//   <div className="calcContainer">
+//     <Display />
+//     <BtnContainer />
+//   </div>
+// );
```

Comment:
- [ ] To make your code clean, kindly remove commented codes from your code!

Reply:
Thanks a lot, I'll do that 😄